### DISCUSSION TITLE
1.19 Warning during REI initialization

### DISCUSSION THREAD

**Discussion by HeatUp0** [shedaniel/RoughlyEnoughItems#920] [Q & A] [answered]
Was trying to load up rei on the latest 1.19 fabric version with a modpack of mine, added the artifacts mod and it prompted me with a warning i cannot get past, an image of the warning is here, I dont know what its asking me to do or how to fix it, anything helps
![image](https://user-images.githubusercontent.com/107514188/173691939-b4585b6f-c2ed-41aa-b54e-d7636ab2068b.png)

**Comment by rdrpenguin04** [comment, marked as answer]
Duplicate of #912
@shedaniel, could you close this?

**Reply by HeatUp0** [reply]
Why close it? There hasent been a proper fix given other than to just delete the mod causing the issue but thats not an option for what im trying to achieve

**Reply by rdrpenguin04** [reply]
Consolidation; I already linked the two issues by posting the issue link, and having only one issue in the tracker makes it easier to manage.

**Reply by HeatUp0** [reply]
Fair point, is it possible to give me a heads up whenever the issue is resolved tho?

**Reply by rdrpenguin04** [reply]
You can set yourself as a watcher on the original issue (which I think is actually #897)
